server/container/guice/cassandra-guice/src/test/java/org/apache/james/DockerCassandraRule.java

```diff
@@ -0,0 +1,92 @@
+/****************************************************************
```

Comment:
I wonder if I should move this part to a separate commit something like "Add Junit rule to start cassandra in a docker what do you think ?"